### ISSUE TITLE
LazyVim distribution is not found as a plugin.

### ISSUE THREAD

**Issue by lkhphuc** [polirritmico/telescope-lazy-plugins.nvim#4]
I am using LazyVim distribtion with this config in `nvim/lua/config/lazy.lua`:
```lua
require("lazy").setup({
  spec = {
    {
      "LazyVim/LazyVim",
      -- dev = true,
      import = "lazyvim.plugins",
      opts = { colorscheme = "catppuccin" }
    },
    { import = "plugins" },
  },
...
```
Using the default option, this plugin does not find LazyVim as a plugin itself, even though it can find the configuration of other plugins defined inside LazyVim.
It's not a big problem since all I want is to check the `options.lua` of LazyVim.
Currently I can do so by selecting a different plugin that is defined by LazyVim then fuzzy search other file in the same local folder.

**Comment by polirritmico** [comment]
Hi @lkhphuc, thanks for reporting the issue and using the plugin!

At first, I consider this out of scope since LazyVim is a configuration and not a plugin per se... But, I understand that it could be very handy in some configurations. So, instead of adding support for LazyVim, NvChad, AstroNvim, etc., I'm going to add an option to allow custom entries from the user and _maybe_ some configuration snippets in the documentation.

Would you consider that approach to be OK for this use case?

**Comment by lkhphuc** [comment]
That would be great.

**Comment by polirritmico** [comment]
Closed by #5
